site/content/en/docs/Guides/metrics.md

```diff
@@ -43,6 +43,7 @@ Follow the [Stackdriver Installation steps](#stackdriver-installation) to see yo
 | agones_gameservers_count                              | The number of gameservers per fleet and status                                                                                                                                              | gauge     |
 | agones_gameserver_allocations_duration_seconds        | The distribution of gameserver allocation requests latencies                                                                                                                                | histogram |
 | agones_gameservers_total                              | The total of gameservers per fleet and status                                                                                                                                               | counter   |
+| agones_gameservers_players_in_game                    | The total number of players connected to gameservers (Only available when [player tracking]({{< relref "player-tracking.md" >}}) is enabled)                                              | gauge     |
```

Comment:
Since docs are published on merge, we'll need you to create two instance of this table, each in `feature` shortcodes, one with the new metric(s) and one without.

That way the new metric will stay hidden until the next release 👍🏻 

See:
https://agones.dev/site/docs/contribute/#within-a-page

Reply:
Instead of duplicating the table, I added the `feature` shortcode within the existing one to hide the new metrics until Agones 1.28.
Let me know if this is suitable or if I should duplicate the table instead.

Reply:
Disregard the last comment. Had to duplicate the table as what I did wasn't working as expected.